wremnants/__init__.py

```diff
@@ -25,5 +26,6 @@
 from .theory_tools import scale_tensor_axes, define_prefsr_vars, moments_to_angular_coeffs
 from .muon_calibration import *
 from .helicity_utils import *
+from .helicity_utils_polvar import makehelicityWeightHelper_polvar
```

Comment:
This might eventually go in helicity_utils, no real need to have an independent helicity_utils_polvar container

Reply:
Either is fine

Reply:
I kind of hate this use of the __init__.py file to import everything else, I would just access it from the submodule directly

Reply:
I agree, In particular I don't think relative imports and "*" imports are good practice

Reply:
ok but what should I edit then? My change or the other preexisting things?

Reply:
I think it is OK if you just edit your change to not add more stuff to the init.py. If we really want to clean up this we can do another PR. My initial comment was on the files, I'm fine with either having separate files or one file for the theory agnostic.